apps/webapp-api/src/perl/MediaWords/Controller/Api/V2/Stories.pm

```diff
@@ -12,7 +12,7 @@ use HTTP::Status qw(:constants);
 use Readonly;
 use Encode;
 
-use MediaWords::Annotator::Store;
+# use MediaWords::Annotator::Store;
```

Comment:
are the changes required here as simple as deleting what i've got commented out at present?

Reply:
Yeah, just delete everything `MediaWords::Annotator` related and the package itself please.